IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ElectricalDeviceCommon/DocPropertySet.xml

```diff
@@ -14,6 +14,11 @@
 		<DocProperty xsi:nil="true" href="InsulationStandardClass_1TFB_0qTyHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="IP_Code_1MhaW0qTyHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="IK_Code_2kpG9_rK93LwNz4pCQcRnH" />
+		<DocProperty xsi:nil="true" href="EarthingStyle_31GzGB0Kz6DfNA40OdlMGX" />
+		<DocProperty xsi:nil="true" href="HeatDissipation_0lf5nUXkv7RfqST9JcFatz" />
+		<DocProperty xsi:nil="true" href="MaximumPower_3GU6eTlpT678UXrRfzKcYW" />
+		<DocProperty xsi:nil="true" href="NominalPowerConsumption_2Yq0U4UVj5mxLmMYOcirrz" />
+		<DocProperty xsi:nil="true" href="NumberOfPowerSupplyPorts_2zCP6ydR933h5ygkKmdVFt" />
```

Comment:
In an IFC modelling sense this would be accessible through the number of unique ports with a POWER system type but if you feel a short hand property is required ok. 

this could also be considered a duplication of NumberOfPoles which is: The number of live lines that is intended to be handled by the device.

Reply:
keep for now